### PR TITLE
Fix some wrong keys in mappings

### PR DESCRIPTION
**Pull request by meew0**
No huge problems, but it would cause annoying messages to be printed to console every time.

### CODE REVIEW

File: src/discordcr/client.cr [hunk 1]
```diff
@@ -94,18 +94,25 @@ module Discord
...
+            handle_dispatch(packet.event_type, packet.data)
+          when OP_RECONNECT
+            handle_reconnect
+          when OP_INVALID_SESSION
+            handle_invalid_session
+          else
+            puts "Unsupported message: #{message}"
+          end
+        rescue ex
+          puts "An exception occurred during message parsing!"
```
Comment (RX14) [inline src/discordcr/client.cr]: Is this really just during message parsing? If a handler raises *any* exception won't it filter up to here? This includes user handlers and caching exceptions.

Comment (meew0) [inline src/discordcr/client.cr]: You're right, I should change it so it only triggers on JSON parse exceptions.

File: src/discordcr/client.cr [hunk 2]
```diff
@@ -109,10 +109,13 @@ module Discord
             puts "Unsupported message: #{message}"
           end
         rescue ex : JSON::ParseException
-          puts "An exception occurred during message parsing!"
+          puts "An exception occurred during message parsing! Please report this."
           puts ex.message
```
Comment (RX14) [inline src/discordcr/client.cr]: Use `inspect_with_backtrace` here too.

File: src/discordcr/client.cr [hunk 3]
```diff
@@ -238,7 +241,14 @@ module Discord
 
     # :nodoc:
     macro call_event(name, payload)
-      @on_{{name}}_handlers.try &.each { |handler| handler.call({{payload}}) }
+      @on_{{name}}_handlers.try &.each do |handler|
+        begin
+          handler.call({{payload}})
+        rescue ex
+          puts "An error occurred in an event handler!"
```
Comment (RX14) [inline src/discordcr/client.cr]: "An exception occurred in a user-defined event handler!"

**Pull request merged**